src/components/Dashboard/TalkDashboard.vue

```diff
@@ -116,7 +116,7 @@ const filteredConversations = computed(() => store.getters.conversationsList.fil
 async function startMeeting() {
 	try {
 		const conversation = await store.dispatch('createGroupConversation', {
-			roomName: conversationName.value ?? t('spreed', 'Meeting'), // TRANSLATORS: Fallback for instant meeting name; it uses user language statically
+			roomName: conversationName.value || t('spreed', 'Meeting'), // TRANSLATORS: Fallback for instant meeting name; it uses user language statically
```

Comment:
Btw the translator hint does not work as the string exists in another location which doesn't have it:
![image](https://github.com/user-attachments/assets/61c85539-3086-492e-a274-9e8a0c829506)